deepchem/models/tests/test_optimizers.py

```diff
@@ -255,13 +262,70 @@ def test_linearCosine_decay_jax(self):
   def test_PieceWise_decay_jax(self):
     """test creating an optimizer with a PeiceWise constant decay to the learning rate"""
     import optax
-    rate = optimizers.PiecewiseConstantSchedule(
-        initial_rate=0.1,
-        boundaries_and_scales={
-            5000: 0.1,
-            10000: 0.1,
-            15000: 0.1
-        })
+    rate = optimizers.PiecewiseConstantSchedule(initial_rate=0.1,
+                                                boundaries_and_scales={
+                                                    5000: 0.1,
+                                                    10000: 0.1,
+                                                    15000: 0.1
+                                                })
     opt = optimizers.Adam(learning_rate=rate)
     jaxopt = opt._create_jax_optimizer()
     assert isinstance(jaxopt, optax.GradientTransformation)
+
+  @pytest.mark.torch
+  def test_KFAC(self):
+    """test creating a KFAC optimizer"""
+    import torch
+    import numpy as np
+
+    np.random.seed(123)
+    """linear layers test"""
```

Comment:
Comments inside functions should start with `#`.  Only docstrings are string literals.